packages/calendar/test/lion-calendar.test.js

```diff
@@ -504,77 +393,6 @@ describe('<lion-calendar>', () => {
         expect(elObj.activeMonthAndYear).to.equal('December 2000');
       });
 
-      it('handles switch to previous month when dates are disabled', async () => {
```

Comment:
This functionality should really still work and be tested.
Any reason for removing it?